lesson02/assignment/css/style copy.scss

```diff
@@ -0,0 +1,58 @@
+// Import partials here
+@import 'base.normalize';
```

Comment:
This file seems to be duplicated along with style.css. Make sure to check for extra copies in the future. Can be frustrating if you end up editing the wrong one and your code doesn't reflect it, etc.